src/Nbody/Sinks.cpp

```diff
@@ -614,6 +614,9 @@ void Sinks<ndim>::AccreteMassToSinks
       for (k=0; k<ndim; k++) sink[s].star->a[k] *= sink[s].star->m;
       //for (k=0; k<ndim; k++) sink[s].star->adot[k] *= sink[s].star->m;
 
+      // Update sink accretion rate
+      //-------------------------------------------------------------------------------------------
+      sink[s].dmdt = macc / timestep;
```

Comment:
Is this valid for both the simple accretion and smoothed accretion?

Reply:
I believe so, this is after macc is calculated for both cases. We don't care about how that mass is doled out from particles in the smooth accretion case, which is what happens afterwards.

Reply:
I think this should actually be macc / (timestep * nstep) (see dt in the smoothed accretion routine)